code/game/objects/effects/spawners/depot_spawners.dm

```diff
@@ -37,14 +37,14 @@
 	name = "33pc trap pizza"
 	icon_state = "pizzabox"
 	loot = list(
-		/obj/item/pizzabox/pepperoni = 7, // Higher weight as a pizza bomb looks like pepperoni by default
-		/obj/item/pizzabox/pizza_bomb/autoarm = 7,
 		/obj/item/pizzabox/firecracker = 1,
 		/obj/item/pizzabox/garlic = 1,
 		/obj/item/pizzabox/hawaiian = 1,
 		/obj/item/pizzabox/margherita = 1,
 		/obj/item/pizzabox/meat = 1,
 		/obj/item/pizzabox/mushroom = 1,
+		/obj/item/pizzabox/pepperoni = 1,
+		/obj/item/pizzabox/pizza_bomb/autoarm = 5,
```

Comment:
Previous one had a 33% chance to be a pizza bomb. I think you want this to be `4` to keep the same percent. (4/(4 + 8)). Can you also move the pizza bomb back up to the top of the list so its more obvious that its different than normal pizzas by not being sorted alphabetically.
```suggestion
		/obj/item/pizzabox/pizza_bomb/autoarm = 4,
```